# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1212419938 14400
#      Mon Jun 02 11:18:58 2008 -0400
# Branch release-3-0-x
# Node ID fe8f87d9133eb02525305baf218079bf125e87fd
# Parent  65e41465c46b27e952b2bcf4ec4610e53052c1c7
mod: delete bogus test

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2008-06-02  John W. Eaton  <jwe@octave.org>
+
+	* general/mod.m: Delete bogus test.
+
 2008-06-02  Jaroslav Hajek <highegg@gmail.com>
 
 	* strings/strcat.m: Add tests.
 
 2008-06-02  Kim Hansen  <kimhanse@gmail.com>
 
 	* strings/mat2str.m: Change is_complex to iscomplex, add tests, add
 	missing ;
diff --git a/scripts/general/mod.m b/scripts/general/mod.m
--- a/scripts/general/mod.m
+++ b/scripts/general/mod.m
@@ -122,9 +122,8 @@ endfunction
 %!assert (mod(uint8([1:5]),uint8(4)),uint8([1,2,3,0,1]))
 %!assert (mod(uint8([1:5]),uint8(0)),uint8([1:5]))
 %!error (mod(uint8(5),int8(4)))
 
 ## mixed integer/real types
 %!assert (mod(uint8(5),4),uint8(1))
 %!assert (mod(5,uint8(4)),uint8(1))
 %!assert (mod(uint8([1:5]),4),uint8([1,2,3,0,1]))
-%!error (mod([1:5],uint8(4)))
